addon/mixins/in-viewport.js

```diff
@@ -215,19 +218,23 @@ export default Mixin.create({
   _bindScrollDirectionListener(sensitivity = 1) {
     assert('sensitivity cannot be 0', sensitivity);
 
-    const $contextEl = get(this, 'scrollableArea') ? $(get(this, 'scrollableArea')) : $(window);
+    const contextEl = get(this, 'scrollableArea') || window;
+    let elem = findElem(contextEl);
 
-    $contextEl.on(`scroll.directional#${get(this, 'elementId')}`, () => {
-      this._debouncedEventHandler('_triggerDidScrollDirection', $contextEl, sensitivity);
+    elem.addEventListener('scroll', () => {
+      this._debouncedEventHandler('_triggerDidScrollDirection', elem, sensitivity);
     });
   },
 
   _unbindScrollDirectionListener() {
     const elementId = get(this, 'elementId');
 
     const context = get(this, 'scrollableArea') || window;
+    let elem = findElem(context);
 
-    $(context).off(`scroll.directional#${elementId}`);
+    elem.removeEventListener('scroll', () => {
+      this._debouncedEventHandler('_triggerDidScrollDirection', elem, get(this, 'viewportScrollSensitivity'));
+    });
```

Comment:
I'm not totally sure of the implications of this.  It wouldn't be good if the removeEventListener matching algorithm didn't match....so maybe I should put the callback as a method on the mixin instead of repeating it in the addEventListener callback.